### ISSUE TITLE
Code removed from output

### ISSUE THREAD

**Issue by Rogdham** [react-syntax-highlighter/react-syntax-highlighter#210]
## Describe the bug

Highlighting the following code as `html` results in the `alert()` part being removed:

```
<script>alert()</script>
```

becomes 

```html
<script></script>
```

### JSX

```jsx
import SyntaxHighlighter from 'react-syntax-highlighter';

const code = `<script>alert()</script>`;
const element = <SyntaxHighlighter language="html">{code}</SyntaxHighlighter>;
```

### Node.js
Let's use [babel](https://babeljs.io/repl) to transcode the following:
```jsx
import React from "react";
import renderer from "react-test-renderer";
import SyntaxHighlighter from "react-test-renderer";

const code = `<script>alert()</script>`;

const tree = renderer
  .create(<SyntaxHighlighter language="html">{code}</SyntaxHighlighter>)
  .toJSON();
console.log(JSON.stringify(tree, undefined, 2));
```

into:
```js
"use strict";

var react = _interopRequireDefault(require("react"));

var reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

var reactSyntaxHighlighter = _interopRequireDefault(
  require("react-syntax-highlighter")
);

function _interopRequireDefault(obj) {
  return obj && obj.__esModule ? obj : { default: obj };
}

var code = "<script>alert()</script>";

var tree = reactTestRenderer.default
  .create(
    react.default.createElement(
      reactSyntaxHighlighter.default,
      {
        language: "html"
      },
      code
    )
  )
  .toJSON();

console.log(JSON.stringify(tree, undefined, 2));
```

When running this script with Node.js (provided react-test-renderer is installed as well), it outputs:
```json
{
  "type": "pre",
  "props": {
    "style": {
      "display": "block",
      "overflowX": "auto",
      "padding": "0.5em",
      "background": "#F0F0F0",
      "color": "#444"
    }
  },
  "children": [
    {
      "type": "code",
      "props": {},
      "children": [
        {
          "type": "span",
          "props": {
            "className": "hljs-tag",
            "style": {}
          },
          "children": [
            "<",
            {
              "type": "span",
              "props": {
                "style": {
                  "fontWeight": "bold"
                }
              },
              "children": [
                "script"
              ]
            },
            ">"
          ]
        },
        {
          "type": "span",
          "props": {
            "className": "null",
            "style": {}
          },
          "children": null
        },
        {
          "type": "span",
          "props": {
            "className": "hljs-tag",
            "style": {}
          },
          "children": [
            "</",
            {
              "type": "span",
              "props": {
                "style": {
                  "fontWeight": "bold"
                }
              },
              "children": [
                "script"
              ]
            },
            ">"
          ]
        }
      ]
    }
  ]
}
```

as you can see, the `alert()` part is replaced with:
```json
        {
          "type": "span",
          "props": {
            "className": "null",
            "style": {}
          },
          "children": null
        },
```

## Expected behavior

The output should still have `alert()` from the input.

## Desktop (please complete the following information):
 - OS: ArchLinux
 - Browser: tested with Firefox 68.0.2 and Chrome76.0.3809.100
 - Version: react-syntax-highlighter 11.0.2

**Comment by Rogdham** [comment]
This is a bug in lowlight: https://github.com/wooorm/lowlight/pull/17 which has been fixed in version 1.12.1 :tada:

@conorhastings could you consider updating the lowlight dependency? :pray:

**Comment by conorhastings** [comment]
@Rogdham currently in flight will make change when i reutrn home

**Comment by conorhastings** [comment]
hmm tests are failing when i updaet the dep, am looking into it tho

**Comment by Rogdham** [comment]
I've given #213 a quick look, and indeed tests seems to be failing (even though the CI seems to be ok).

I'm not sure how much of it is due to lowlight now rendering code that was deleted before (i.e. what this issue is about) or just due to other changes.

**Comment by Rogdham** [comment]
Hello @conorhastings :wave:, did you make any progress on the issue?

**Comment by Rogdham** [comment]
It seems that even though #213 is merged, the issue is still not fixed since:
 1. this package needs `react-syntax-highlighter-virtualized-renderer^1.0.3`
 2. …which needs `react-syntax-highlighter^5.1.2`
 3. …and since it's an old version, it still needs an old version of lowlight.

Maybe there is a better way to specify such a _circular dependency_?

**Comment by Rogdham** [comment]
Thinking back about this, `react-syntax-highlighter-virtualized-renderer` is only a dozen of lines long, so it would make sense to merge that code into the `react-syntax-highlighter` maybe?

If my understanding is correct, both packages depends on each other, so all their dependencies would be installed anyways. Obviously, I'm missing why you (@conorhastings) split the code into to repositories, so feel free to argue!

**Comment by Rogdham** [comment]
In the meanwhile, a workaround is to use [Yarn's resolution feature](https://yarnpkg.com/en/docs/selective-version-resolutions) by adding the following to `package.json`:

```json
  "resolutions": {
    "react-syntax-highlighter": "12.2.1"
  },
```

**Comment by simmerer** [comment]
[We're using `lowlight=1.14.0` now](https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/package.json#L11), so this is resolved in `react-syntax-highlighter` 13.0.0+

Try it here in the updated demo: https://react-syntax-highlighter.github.io/react-syntax-highlighter/demo/prism.html
